ui-components/contexttoolbar.md

```diff
@@ -19,29 +19,16 @@ A context toolbar is registered by calling the `addContextToolbar` API in the re
 | `scope` | This controls whether the predicate is a `node`-based predicate, or an `editor`-based predicate. See context toolbar proirity section below, for more details. |
 | `items` | A list of strings which represent either a registered toolbar button, or a registered context form launcher. |
 
-### Launching a context toolbar programmatically
-
-There is an `editor` event called `contexttoolbar-show` that can be fired to show a context toolbar at the current selection. The event takes a parameter `toolbarKey` which specifies the name of the registered context form or context toolbar to show.
-
-### Context toolbar priority
-
-{% include context/priority.md %}
-
-### Positioning context toolbars
-
-There are three options for positioning are: `selection`, `line`, or `node`.
-
-* A `selection` position will place the context toolbars above or below the current selection, centred if possible.
-
-* A `line` position will place the context toolbars to the right (or left in Right-to-Left languages) of the current selection.
-
-* A `node` position will place the context toolbars relative to the bounds of a node (e.g. a table or image). It applies to a selected node that does not match the selection due to CSS properties (like float).
-
-
+{% include context/positioning.md %}
 
 ### Example configuration
 
 This example shows how the quickbars plugin adds the standard selection context toolbar and an example of a custom toolbar for image alignment. The context toolbar will show whenever any content is selected.
 
 {% include codepen.html id="context-toolbar" height="600" tab="js" %}
 
+### Launching a context toolbar programmatically
```

Comment:
I thought we weren't going to be documenting this and it was meant to be an internal-only thing for now?

Reply:
@metricjs you might know more on this? however, I remember being told it shouldn't be documented when I first started.

Reply:
We can remove it if needed. It has been in the docs for more than 2 years: https://github.com/tinymce/tinymce-docs/blame/88f95d130a77a2bfce3b128a266c21a22d2d8ff9/ui-components/contexttoolbar.md#L24

Reply:
Ignore then I guess :shrug:

Reply:
after discussion, deciding to leave the content in.